src/archival/utils.rs

```diff
@@ -25,7 +27,11 @@ pub async fn get_first_id_to_start_notifier_from(pool: PgPool) -> Option<i32> {
 }
 
 /// Updates a row in `internet_archive_urls` table with the `job_id` response received from `Wayback Machine API` request, and marks `is_saved` true.
-pub async fn update_internet_archive_urls(pool: &PgPool, job_id: String, id: i32) {
+pub async fn update_internet_archive_urls_with_job_id(
```

Comment:
The name of this function is overly complicated. Just go simpler:
```suggestion
pub async fn set_archive_IA_url(
```